sessions/backend-update-and-delete/products/pages/_app.js

```diff
@@ -0,0 +1,15 @@
+import { SWRConfig } from "swr";
+import GlobalStyle from "../styles";
+
+const fetcher = (url) => fetch(url).then((r) => r.json());
+
```

Comment:
I have seen a few times that the response was abbreviated with "res", but just "r" feels a bit too short to me 🤔 
As we tell our students to not use abbreviations, I think we should stick to that in challenges as well. If this is common practice (shown in official documentations), I think we could make an exception. 🤷🏼